firefox-ios/Client/Frontend/Fakespot/FakespotState.swift

```diff
@@ -10,6 +10,10 @@ struct FakespotState: ScreenState, Equatable {
     var sidebarOpenForiPadLandscape: Bool
     var currentTabUUID: String
     var expandState: [String: ExpandState]
+    var isBottomSheetDisplayed: Bool
+    var exposedToAdsEvent: Bool
+    var notExposedToAdsEvent: Bool
```

Comment:
I'm curious why we need a notExposedToAdsEvent. Why can't it be just !exposedToAdsEvent.

Reply:
These are two distinct events, each with its own state. By treating them as such, we prevent the redundant recording of the same telemetry event when the review checker is closed and reopened within the same tab, according to the specs.